src/org/omegat/gui/main/ProjectUICommands.java

```diff
@@ -649,7 +650,11 @@ private static void projectOpenImpl(File projectRootFolder) {
                             backup.getName());
                     Core.getProject().saveProjectProperties();
                 } else if (FileUtil.getRecentBackup(projectFile) == null) {
-                    FileUtil.backupFile(projectFile);
+                    File backup = new File(projectRootFolder,
+                            String.format("%s.%s%s", OConsts.FILE_PROJECT,
+                                    new SimpleDateFormat("yyyyMMddHHmm").format(new Date(projectFile.lastModified())),
+                                    OConsts.BACKUP_EXTENSION));
```

Comment:
Here is copy of a similar lines of code with FileUtil class.
Are any ways to update `FileUtil.backupFile`  method? 
My idea is adding `FileUtil.getBackupFilename` utility method and `FileUtil.backupFile` and here share a same logic.

Reply:
That's a good idea, thanks for the suggestion.